service.go

```diff
@@ -90,7 +90,7 @@ func (s *service) createHTTPComponent() (Component, error) {
 		}
 	}
 	port = strconv.FormatInt(portVal, 10)
-	log.Infof("creating default HTTP component at port %s", port)
+	log.Debugf("creating default HTTP component at port %s", port)
```

Comment:
same as above

Reply:
There is already a requested change at the HTTP component level that logs this. If we move forward with the changes we should have only one of those logs.

Reply:
ok, that makes sense. Missed the other one